src/main/java/com/minecolonies/coremod/util/StructureWrapper.java

```diff
@@ -365,15 +398,15 @@ else if ((structureBlock instanceof BlockStairs && structureBlockState == worldB
         }
 
         final Template.EntityInfo entityInfo = structure.getEntityinfo(this.getLocalPosition());
-        if(entityInfo != null)
+        if (entityInfo != null)
         {
             return false;
             //todo get entity at position.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)